### DISCUSSION TITLE
Export action: export the same column twice/multiple times in different formats

### DISCUSSION THREAD

**Discussion by bafforosso** [filamentphp/filament#11229] [Help]
### Package

Actions

### Package Version

v3.2.23

### How can we help you?

I wrote an Exporter for a model and I want to export a column under two different formats : Human readable and machine readable.

It's basically an aggregate of a sum of duration in seconds, so I would like to have it as a first column in `H:i:s` format (which is what the `pretty_hours()` helper does) and the next column the same value but in seconds.

```PHP
ExportColumn::make('tracked_times_sum_duration')
   ->label('Total Time Spent (Human Readable)')
   ->formatStateUsing(function ($state) {
      return pretty_hours($state);
   }),
ExportColumn::make('tracked_times_sum_duration')
   ->label('Total Time Spent (seconds)'),
```

Both of my `ExportColumn` are working perfectly independently, but if I put both in the same Exporter then it's not working since the `$name` in `ExportColumn::make($name)` makes Filament all confused:
- first `ExportColumn` name showed twice in the modal for column selections, modifying one modify both columns header simultaneously.
- only the second `ExportColumn` is there in the export file.

Is there a way to easily do what I want or a hint to solve the problem?

Thank you very much in advance

**Comment by micraux** [comment]
Hello,
I have the same use case.
The problem is that you have twice the `make('tracked_times_sum_duration')`.
So, try this for your second one:
```
ExportColumn::make('tracked_times_sum_duration_2')
   ->label('Total Time Spent (seconds)')
   ->state(fn (Yourmodel $record): int => $record->tracked_times_sum_duration),
```

**Reply by micraux** [reply]
Hello @bafforosso,
Did this solve your problem?
If so, you can close this thread...
